src/components/CreateSectionForm.vue

```diff
@@ -0,0 +1,241 @@
+<template>
+  <div>
+    <h2>Create/Edit Section</h2>
+    <br>
+
+    <form @submit.prevent>
+      <!-- Put fields & buttons with their labels here -->
+        <div >
+          <ValidationProvider name="Section Tag" rules="required" v-slot="{ errors }">
+            <label for="sectionTags">Section Tag</label><br>
+            <input list= "sectionTags" name="sectionTag" type="text" style="width:300px;" v-model="sectionTag">
+            <datalist id=sectionTags>
+              <!--Example, should import user's sections from db-->
+              <option value=ADAM-ECE3221-FRO1B-LEC></option>
+            </datalist>
+            <span v-if="errors.length" class="error">{{ errors[0] }}</span>
+          </ValidationProvider>
+          <br><br>
+          <button 
+            style="width:300px;"
+            v-on:click="$.createSection()"
```

Comment:
In the vue attributes you can jut put the name of the function directly.
```suggestion
            v-on:click="createSection"
```